code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -3,25 +3,13 @@
 	density = 1
 	anchored = 1
 	icon = 'icons/obj/cult.dmi'
-	var/cooldowntime = 0
-	var/health = 100
-	var/death_message = "<span class='warning'>The structure falls apart.</span>" //The message shown when the structure is destroyed
-	var/death_sound = 'sound/items/bikehorn.ogg'
```

Comment:
Aren't health and death sounds/messages common to all cult structures, including pylons